app/app.go

```diff
@@ -272,17 +268,26 @@ func New() *cli.App {
 }
 
 func download(c *cli.Context, videoURL string) error {
+	errcontinue := c.Bool("errorcontinue")
+
+	cookie, err := utils.ReadCookie(c.String("cookie"))
+	if err != nil {
+		return nil
+	}
```

Comment:
We can make the `cookie` as a parameter, so there's no need to parse it repeatedly.